companies/123greetings.json

```diff
@@ -1,22 +1,23 @@
 {
-    "slug": "123greetings",
+    "slug": "123greeting2s",
     "relevant-countries": [
-        "all"
+        "all",
+        "dong"
     ],
     "categories": [
         "entertainment"
     ],
     "name": "123Greetings.com Inc.",
-    "address": "255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
+    "address": "123Greetings.com Inc.\n255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
```

Comment:
🚫 **[recordlint]** <[name-in-address](https://github.com/datenanfragen/data#addresses)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record includes `name` in the `address`.
```suggestion
    "address": "255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
```